shared/middleware/files_api.rb

```diff
@@ -460,7 +460,7 @@ def copy_file(endpoint, encrypted_channel_id, filename, source_filename)
   end
 
   #
-  # GET /v3/files/<channel-id>
+  # GET /v3/files/<channel-id>?version=<version-id>
```

Comment:
Elsewhere, like the delete all operation, this project version parameter is named/documented differently.

```
   #
   # DELETE /v3/files/<channel-id>/*?files-version=<project-version-id>
   #
   # Delete all files.
   #
```

Reply:
At this point, we don't support `GET` based on `files-version` - we could do that, but it requires an extra network request (first, to fetch the mainfest, then, to fetch the file). And based on the way we get the project, the caller already has access to the file version (and the overall project's files-version) so it seems smarter to just jump directly to that version of the file. We support it in other places to keep the manifest file generation in check (this is important for cases where we modify the project: rename, delete, put a file, etc.)

Reply:
But this particular endpoint `GET /v3/files/<channel-id>` isn't getting one particular file, it's listing files of the project - so really, the manifest is the only file it should be reading.  Doesn't that mean `files-version` is the one we want to use here?

Reply:
So, the API and the docs both require `version` on this endpoint (currently). The reason for this is that when referring to the root of the project (aka the manifest), the `version` is the `files-version` used in other APIs. For API symmetry, we should probably accept `files-version` as well here. But for now, we don't and the docs are correct. So, I'm not inclined to make further changes right now.